CleanProFinder/CleanProFinder.Server/Services/Implementations/ValidationService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using CleanProFinder.Server.Services.Interfaces;
+using CleanProFinder.Shared.Errors.Base;
+using CleanProFinder.Shared.ServiceResponseHandling;
+using FluentValidation;
+using System.Reflection;
+
+
+namespace CleanProFinder.Server.Services.Implementations
+{
+    public class ValidationService : IValidationService
+    {
+        public async Task<ServiceResponse> ValidateAsync<T>(T item)
+        {
+            var validatorType = Assembly.GetExecutingAssembly().GetTypes()
```

Comment:
Do not forget that we can store our validators in not same project as where this service is stored. One way to fix it is to describe assemblies in constructor. Constructor will be used when injecting in DI container